.gitlab-ci.yml

```diff
@@ -245,6 +245,12 @@ build:base+async:
     COQ_EXTRA_CONF: "-native-compiler yes -coqide opt"
     COQUSERFLAGS: "-async-proofs on"
 
+build:quick:
+  <<: *build-template
+  variables:
+    COQ_EXTRA_CONF: "-native-compiler no"
```

Comment:
Do we want -native-compiler yes here?

Reply:
I don't think so, it does not compile opaque proofs hence we would just slow down the build imo

Reply:
OK, let's merge. If someone wants native compiler they can always make a subsequent PR.